pkg/lib/operation/project/sync/pull/operation.go

```diff
@@ -118,3 +126,17 @@ func Run(ctx context.Context, projectState *project.State, o Options, d dependen
 
 	return nil
 }
+
+func ignoreConfigsAndRows(projectState *project.State) {
+	if len(projectState.IgnoredConfigRows()) > 0 {
+		for _, v := range projectState.IgnoredConfigRows() {
+			v.SetRemoteState(nil)
+		}
+	}
+
+	if len(projectState.IgnoredConfigs()) > 0 {
```

Comment:
and this one

Reply:
[ae0cdbe](https://github.com/keboola/keboola-as-code/pull/2136/commits/ae0cdbebc3d40eb6f5c644e1c3c260a1c5d56030)